jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST

```diff
@@ -50,6 +50,9 @@ cp "${DATA}/soca_gridspec.nc" "${COMOUT}/${CDUMP}.t${bcyc}z.ocngrid.nc"
 mkdir -p "${COMOUT}/logs"
 cp "${DATA}/*.out" "${COMOUT}/logs"
 
+# Copy var.yaml
+cp "${DATA}/var.yaml" "${COMOUT}"
```

Comment:
@AndrewEichmann-NOAA , depending on what the cycle runs, we can get up to ~ a dozen yamls ... We should copy all of them to a dedicated directory.
something like this : ```cp $DATA/*.yaml $COMOUT/yaml```

Reply:
@guillaumevernieres Do you expect them all to me in `$DATA`?

Reply:
And not in any subdirectories? They all are now, so I'll just go with that

Reply:
yes, the one we generate will be in $DATA.

Reply:
when `*` is put inside of `" "`, it becomes a character and ceases to be a wildcard.